vulkan/vulkan_backbuffer.h

```diff
@@ -7,43 +7,56 @@
 
 #include <array>
 
+#include "flutter/vulkan/vulkan_command_buffer.h"
+#include "flutter/vulkan/vulkan_handle.h"
+#include "lib/ftl/compiler_specific.h"
 #include "lib/ftl/macros.h"
-#include "vulkan_proc_table.h"
-#include "vulkan_handle.h"
+#include "third_party/skia/include/core/SkSize.h"
+#include "third_party/skia/include/core/SkSurface.h"
 
 namespace vulkan {
 
 class VulkanBackbuffer {
```

Comment:
Since usage of the fences/semaphores/command-buffers is not encapsulated in this class, please document the intended usage by callers.

Again, I can't see anywhere in this CL that VulkanBackbuffer is instantiated (again, I would expect to see changes to the instantiation site(s), due to the change in the signature of the constructor).  Am I missing something?